data/scenarios/Challenges/2048.yaml

```diff
@@ -4,13 +4,8 @@ author: Brent Yorgey
 description: Make 2048!
 objectives:
   - goal:
-    - OK, OK, it's not really the same as the classic "2048" game.
-      However, your goal is still to make 2048!  You start with a `1`
-      which regrows immediately when
-      it is harvested, so if you plant it, you can get as many as you want.  Your
-      job is to combine `1`s in order to make a `2048`
-      entity.
-    - "Hint: the `format` command can turn numbers into strings!"
+      - OK, OK, it's not really the same as the classic "2048" game. However, your goal is still to make 2048!  You start with a `1` which regrows immediately when it is harvested, so if you plant it, you can get as many as you want.  Your job is to combine `1`s in order to make a `2048` entity.
```

Comment:
I do not have that wide monitor.

Reply:
I noticed that the [Python implementation of `yq`](https://kislyuk.github.io/yq/) has a `--width` option to set line wrap for string literals.  The concern would be if it also ends up wrapping swarm-lang code.  I may have to research a bit more.

Reply:
Swarm lang ignores extra whitespace so that should not break.

Wrapping maps would be a problem, but those should already use the raw format.

Reply:
I tried the Python implementation of `yq` with mixed results:

1. The existing line wraps in `description` and `goal` keys are preserved
2. Quotes are removed from most keys when allowed, e.g.
    ```
    -    'λ': [grass, lambda]
    +    λ: [grass, lambda]
    ```
    but not
    ```
     ' ': [ice, water]
    ```
3. All comments are stripped from the `.yaml` file

Comments right now are fairly pervasive, e.g. in [this file](https://github.com/swarm-game/swarm/blob/9be30c5450f87f96834eb157dac89e065a64a215/data/scenarios/Testing/479-atomic-race.yaml#L1-L16).

An argument could be made that any information that we care about in the `.yaml` file should be programmatically extractable, i.e. assigned as a keyed value within the tree structure of the document.  If you buy that, we should create a new toplevel key called "developer-notes" or something, and move the toplevel `#` comments there.